### ISSUE TITLE
Hong Kong and Macao

### ISSUE THREAD

**Issue by uiharu-s** [nextcloud/privacy#108]
Under "Where is your data?", Hong Kong and Macau are not options for the server location.

Although Hong Kong and Macau are part of the country China (People's Republic of China), they are "Special Administrative Regions" that enforce their own laws, including those concerning personal data privacy and law enforcement requirements, which are extremely different from those of the Mainland of China.

So it should make sense to add those two places as options. However, if we list them as "countries", it would attract heated arguments among the people of these places. So I would also recommend to change "Please select a country" to "Please select a region".

To summarise, I'd like to recommend:
- change "Please select a country" to "Please select a region"
- add "Hong Kong" as a region option
- add "Macao" as a region option

**Comment by brentybh** [comment]
Yes, I support this. Opening a website in Mainland China needs to get an ICP license but not in Hong Kong/Macau

**Comment by jamesstout** [comment]
Hmm, I started adding Hong Kong SAR and Macau (Macao?) ... is it also known as Macau SAR?

Need translations, and an updated Map.vue (from https://github.com/flekschas/simple-world-map) 

Anyone know SVG?

Also ... 

![Screen Shot 2019-07-27 at 7 24 49 PM](https://user-images.githubusercontent.com/1576190/61994001-804bf780-b0a7-11e9-9bc8-77e2db549ea0.png)

potential: 💥

**Comment by brentybh** [comment]
@jamesstout Yes, Macao, Macau, Macau SAR, Macao SAR are same
"Macao" is used in English and in Portuguese it will be called "Macau"
[The government official website](https://www.gov.mo/en/) is using "Macao SAR"

**Comment by georgehrke** [comment]
> Anyone know SVG?

@jamesstout That's indeed the biggest problem here. Due to the AGPL, we can't just use any CC licensed SVG. Certain CC flavours like no derivates are incompatible with AGPL.

If you find a good way to update the current SVG I'd be happy to merge a pull-request! :)

**Comment by georgehrke** [comment]
Duplicate of #241
